README.md

```diff
@@ -1,2 +1,14 @@
 # hw01
-Repository for homework 1
+Homework 1 from Donghyun Kang
+
+<img src="https://raw.githubusercontent.com/nerdizzyz/hw01/master/KANG_DONGHYUN.jpg">
```

Comment:
Could have embedded an image using the following Markdown syntax: ![](image file)

Reply:
thanks!